.github/workflows/build.wheel.sh

```diff
@@ -19,6 +19,11 @@ if [[ "$#" -gt 0 ]]; then
   PYTHON_VERSION="${1}"
   shift
 fi
+
+if [[ "$PYTHON_VERSION" == "python3.8" ]]; then
+  echo "TODO: Python 3.8 test is not supported yet as dependency might not be available"
```

Comment:
Not available where? GitHub Actions?

Reply:
@terrytangyuan Python 3.8 is available in GitHub Actions, and we could build 3.8 wheel. However, all the testing dependencies, like arrow, google cloud python wheels, may not be available as we pined to a specific version (old version) that does not have 3.8. So the test will fail.

We could gradually upgrade those test dependencies to newer version (if 3.8 is available), then we can re-enable python 3.8 tests.